nixos/lib/make-iso9660-image.sh

```diff
@@ -100,6 +100,7 @@ done
 mkdir -p $out/iso
 
 xorriso="xorriso
+ -boot_image any gpt_disk_guid=`uuid -v 5 daed2280-b91e-42c0-aed6-82c825ca41f3 $out | tr -d -`
```

Comment:
`$()` substitution is generally preferred:

```suggestion
 -boot_image any gpt_disk_guid=$(uuid -v 5 daed2280-b91e-42c0-aed6-82c825ca41f3 $out | tr -d -)
```